source/dataplaneapi/app.py

```diff
@@ -18,7 +18,15 @@
 import datetime
 import base64
 
-patch_all()
+def is_aws():
```

Comment:
Only allow xray to patch modules if running on lambda. This allows the local unit tests to properly mock modules.